sdcm/utils/docker_remote.py

```diff
@@ -100,7 +100,7 @@ def check_spot_termination(self):
 
     @property
     def region(self):
-        return "docker"
+        return "eu-north-1"
```

Comment:
why?

Reply:
since now the logic of c-s is calling the newly added function for getting rack, this region needs to be align with the loader for it to work.
so both parts of the test needs to be in sync for that logic to work correctly